### PR TITLE
Fix SRKb logo and remove flip token text

### PR DESCRIPTION
**Pull request by aldrickb**
image
image

### CODE REVIEW

**Review by clowreed — APPROVED** [review]
Flip top!

**Review by Pseudoman21 — APPROVED** [review]
Corrected!

File: src/pages/Pool/index.tsx [hunk 1]
```diff
@@ -118,9 +118,11 @@ export default function Pool() {
                     {hasV1Liquidity ? 'Migrate now.' : TranslateString(108, 'Import it.')}
                   </StyledInternalLink>
                 </Text>
+                {/*
```
Comment (ricoz) [inline src/pages/Pool/index.tsx:121]: Remove na lang wag comment out para di pa din ma view sa source

**Review by ricoz — CHANGES_REQUESTED** [review]
Remove na lang unless di man ma-appear yung commented out sa final source code?

File: src/pages/Pool/index.tsx (hunk 1 above)
Comment (aldrickb) [reply src/pages/Pool/index.tsx:121]: sigi sir, updating

**Review by medardm — APPROVED** [review]
MIRRORED

**Review by ricoz — APPROVED** [review]
Hidden!

**Pull request merged**